x/application/keeper/msg_server_delegate_to_gateway.go

```diff
@@ -2,64 +2,64 @@ package keeper
 
 import (
 	"context"
+	"fmt"
 
 	sdkerrors "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-
 	"github.com/pokt-network/poktroll/x/application/types"
 )
 
 func (k msgServer) DelegateToGateway(goCtx context.Context, msg *types.MsgDelegateToGateway) (*types.MsgDelegateToGatewayResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
 	logger := k.Logger(ctx).With("method", "DelegateToGateway")
-	logger.Info("About to delegate application to gateway with msg: %v", msg)
+	logger.Info(fmt.Sprintf("About to delegate application to gateway with msg: %v", msg))
```

Comment:
Perhaps we should consider adding a polylog implementation that wraps the cosmos-sdk logger. :thinking: This is kinda gross to ready by comparison.

Reply:
@Olshansk can you elaborate on the need for this change, was the logger not treating the first arg like a format string?

Reply:
> Perhaps we should consider adding a polylog implementation that wraps the cosmos-sdk logger. 🤔 This is kinda gross to ready by comparison.

No strong opinion on this. If you believe it'll be beneficial, can you create a ticket and put it on the backlog for a few iterations forward?

Reply:
> can you elaborate on the need for this change, was the logger not treating the first arg like a format string?

I created this diff for example purposes:

```diff
-       logger.Info(fmt.Sprintf("inserted claim for supplier %s with primaryKey %s", claim.SupplierAddress, primaryKey))
+       logger.Info("inserted claim for supplier %s with primaryKey %s", claim.SupplierAddress, primaryKey)
+       // logger.Info(fmt.Sprintf("inserted claim for supplier %s with primaryKey %s", claim.SupplierAddress, primaryKey))
```

Before:

```bash
9:38PM INF inserted claim for supplier pokt19a3t4yunp0dlpfjrp7qwnzwlrzd5fzs2gjaaaj with primaryKey 5f84503087b3159ba59d7fbaaa3a1cfd5a9603e384054d35ca0effa0ccd616d7/pokt19a3t4yunp0dlpfjrp7qwnzwlrzd5fzs2gjaaaj/ method=InsertClaim module=x/supplier
```

After:
```bash
9:53PM INF inserted claim for supplier %s with primaryKey %s method=InsertClaim module=x/supplier pokt19a3t4yunp0dlpfjrp7qwnzwlrzd5fzs2gjaaaj=bd3efa3825a303fee8ceef27280483da31f9073f50766ccfd10257c292bc94a7/pokt19a3t4yunp0dlpfjrp7qwnzwlrzd5fzs2gjaaaj/
9:53PM INF indexed claim for supplier pokt19a3t4yunp0dlpfjrp7qwnzwlrzd5fzs2gjaaaj with primaryKey bd3efa3825a303fee8ceef27280483da31f9073f50766ccfd10257c292bc94a7/pokt19a3t4yunp0dlpfjrp7qwnzwlrzd5fzs2gjaaaj/ method=InsertClaim module=x/supplier
9:53PM INF indexed claim for supplier pokt19a3t4yunp0dlpfjrp7qwnzwlrzd5fzs2gjaaaj at session ending height 32 method=InsertClaim module=x/supplier
```

Reply:
Adding a screenshot as an extra datapoint. Note that the `keypals` in Cosmos' `logger.Info` are not msg formatting args, but actual `key=value` params added at the end.


![Screenshot 2023-12-12 at 2 03 06 PM](https://github.com/pokt-network/poktroll/assets/1892194/aed5b566-1580-45e8-93af-2f8470af615d)